message-transport/pkg/transport/metrics/kafka_over_kafka/package.go

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2018-Present the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// kafka_over_kafka emits metrics from Kafka producers and consumers to Kafka (acting as a "metrics collector").
```

Comment:
isn't go convention to use `doc.go` as the filename for this?

Reply:
Changed to `doc.go`, in two places.